.github/workflows/test.yml

```diff
@@ -9,7 +9,14 @@ on:
     branches:
       - main
       - master
+  workflow_dispatch:
```

Comment:
How is this useful?

Reply:
Wow, this is an artifact of my ci debug process. As you probably know. debugging CI is a nightmare, so `workflow_dispatch` allows you to run CI 'on-click', from Actions menu. You can push any number of commits to the branch and run CI when you think it is needed, not on plain 'push'.
I found that simply adding my branch to 'push' list is enough to debug CI.

Reply:
Oh, thanks, this can be helpful in some cases. Glad to know the feature!

>  I found that simply adding my branch to 'push' list is enough to debug CI.
Yeah, this is the way I always do.